llvm/test/tools/llvm-foreach/retain-return-val.c

```diff
@@ -18,6 +18,6 @@
 // RUN: chmod 777 %t2.sh
 // RUN: %t2.sh
 // RUN: FileCheck < %t.res %s
-// CHECK: Content of first file
-// CHECK: Content of second file
+// CHECK-DAG: Content of first file
```

Comment:
Why do we need `-DAG` here?

Reply:
No need actually, thanks for noticing. This was done to fix test failure when 4 threads were enabled by default.